src/backend/marsha/core/xapi.py

```diff
@@ -0,0 +1,106 @@
+"""XAPI module."""
+from datetime import datetime, timezone
+import re
+import uuid
+
+import requests
+
+
+class XAPI:
+    """The LTI object abstracts an LTI launch request.
+
+    It provides properties and methods to inspect the launch request.
+    """
+
+    def __init__(self, video, statement, lti_user):
```

Comment:
IMO when instanciating this class, we should configure which LRS we will query with which credentials.